src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/NomisEventOutcomeMapper.kt

```diff
@@ -1,9 +1,12 @@
 package uk.gov.justice.digital.hmpps.whereabouts.services
 
+import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.whereabouts.model.AbsentReason
 import uk.gov.justice.digital.hmpps.whereabouts.model.AbsentReason.*
 
-class NomisEventOutcomeMapper {
+
+@Service
+open class NomisEventOutcomeMapper {
   companion object {
```

Comment:
Is this open for future inheritance?

Reply:
It's required for mocking, basically says it can be override. By default it's final.

Reply:
Makes sense, I thought it would have something to do with testing.